am-lib/src/main/java/uk/gov/hmcts/reform/amlib/AccessManagementService.java

```diff
@@ -122,4 +124,17 @@ public FilterResourceResponse filterResource(String userId, String resourceId, J
 
         return null;
     }
+
+    public Map<JsonPointer, Set<Permission>> getRolePermissions(String serviceName, String resourceType,
+                                                                String resourceName, String roleName) {
+
+        RoleBasedAccessRecord roleBasedAccess = jdbi.withExtension(AccessManagementRepository.class,
```

Comment:
There can be many records in DB. Current implementation only handles one record.

Reply:
👍